sigstore/sign.py

```diff
@@ -20,7 +20,7 @@
 ```python
 from pathlib import Path
 
-from sigstore.sign import Signer
+from sigstore.sign import Signer, SigningContext
```

Comment:
Nit: that means this import is no longer necessary, I believe 🙂 

```suggestion
from sigstore.sign import SigningContext
```